java/test/behaviour/typeql/language/insert/BUILD

```diff
@@ -55,7 +55,7 @@ typedb_behaviour_java_test(
     runtime_deps = [
         "//java/test/behaviour/config:parameters",
     ],
-    size = "large",
+    size = "enormous",
```

Comment:
Why has this gotten larger? Are the tests slower?

Reply:
There was a TL at Factory (900 s).

Reply:
How close was it to time out before we changed to the rust-wrapped client? We should always know the root cause of why we're increasing test timeouts

Reply:
When the previous PR was tested on Factory, this test lasted for 830 s. The last attempt for this PR took 757 s. So it is close to 900 and sometimes occasionally take longer.

Reply:
Hmm ok, we can make it larger